specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11/examples/SearchServiceListIndexers.json

```diff
@@ -0,0 +1,67 @@
+{
+    "parameters": {
+        "searchServiceName": "myservice",
+        "api-version": "2017-11-11"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "value": [
+                    {
+                        "name": "myindexer",  
+                        "description": "a cool indexer",  
+                        "dataSourceName": "mydocdbdatasource",  
+                        "targetIndexName": "orders",  
+                        "schedule": {
+                            "interval": "PT1H",
+                            "startTime": "2015-01-01T00:00:00Z"
+                        },  
+                        "parameters": {
+                            "maxFailedItems": 10,
+                            "maxFailedItemsPerBatch": 5
+                        },
+                        "fieldMappings": [],
+                        "disabled": false  
+                    },
+                    {
+                        "name": "myotherindexer",  
+                        "description": "another cool indexer",  
+                        "dataSourceName": "myblobdatasource",  
+                        "targetIndexName": "orders",  
+                        "schedule": { },  
```

Comment:
"interval" is required here.

Reply:
I think the example is meant to show that the schedule itself is not required. I'll remove it.